src/helpers/document-of.ts

```diff
@@ -0,0 +1,5 @@
+import { RenderResult } from '@testing-library/react';
+
+export function documentOf(result: RenderResult): Document {
+	return result.container.ownerDocument;
```

Comment:
Eens zien of deze echt nodig is aangezien de container altijd uw hoogste parent html element teruggeeft